src/client/components/MealMenu/MealCards.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./MealCards.css";
+
+const MealCards = ({ meal }) => {
+  const dateFormat = new Intl.DateTimeFormat("en", {
```

Comment:
Could define this outside of the component, to avoid redeclaring it on every render.